test/constraint.jl

```diff
@@ -614,6 +614,25 @@ function test_Model_list_of_constraint_types(::Any, ::Any)
         (Vector{QuadExpr}, MOI.RotatedSecondOrderCone)])
 end
 
+function test_Model_dual_start(::Any, ::Any)
+    model = Model()
+    @variable(model, x)
+    con = @constraint(model, 2x <= 1)
+    # scalar
+    @test dual_start_value(con) === nothing
+    set_dual_start_value(con, 2)
+    @test dual_start_value(con) == 2.0
+    set_dual_start_value(con, nothing)
+    @test dual_start_value(con) === nothing
+    # vector
```

Comment:
Prefer breaking these into separate functions. The smaller the test function, the better.